pkgs/development/python-modules/pywebcopy/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pywebcopy";
+  version = "7.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-H10GiPyAvX6UVM5by4TqW+z6tcwPqskMnTML3BWJdVU=";
+  };
+
+  postPatch = ''
+    touch requirements.txt
+  '';
+
+  build-system = with python3Packages; [ setuptools ];
```

Comment:
use setuptools from callPackage arguments

Reply:
What do you mean? How can I implement it?